ecl/dns/v2/zones/testing/requests_test.go

```diff
@@ -113,12 +113,12 @@ func TestUpdateDNSZone(t *testing.T) {
 
 		w.WriteHeader(http.StatusOK)
 		w.Header().Add("Content-Type", "application/json")
-		fmt.Fprintf(w, UpdateZoneResponse)
+		fmt.Fprint(w, UpdateZoneResponse)
 	})
 
 	description := descriptionZone1Update
 	ttl := 600
-	masters := []string{}
+	masters := make([]string, 0)
```

Comment:
It is usually recommended to declare variables in a nil slice. 
https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices
However, it is required an empty slice instead of a nil slice here. Therefore an empty slice is declared with "make" explicitly in order to resolve the warning.